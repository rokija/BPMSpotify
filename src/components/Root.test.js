import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Root from './Root';

describe('Root', () => {
    const initProps = {
        store: {},
        history: {}
    };
    const wrapper = shallow(<Root {...initProps} />);

    it('renders Root component with pathname editProfile', () => {
        expect(wrapper.length).toEqual(1);
    });

    it('matches snapshot', () => {
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
