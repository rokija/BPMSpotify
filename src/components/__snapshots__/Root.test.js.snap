// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root matches snapshot 1`] = `
<Provider
  store={Object {}}
>
  <ConnectedRouter
    history={Object {}}
  >
    <App />
  </ConnectedRouter>
</Provider>
`;
